src/Groups/GroupDetails/GroupDetailsContainer.js

```diff
@@ -67,13 +68,17 @@ const GroupDetailsContainer = ({
     return <LoadingPane onClose={onClose} />;
   }
 
-  const fiscalYears = get(resources, ['currentFiscalYears', 'records'], [])
+  const fiscalYear = get(resources, ['currentFiscalYears', 'records'], [])
     .map(({ code: fyCode }) => fyCode).join(', ');
+  const fiscalYears = get(resources, ['currentFiscalYears', 'records'], []);
+  const funds = get(resources, ['funds', 'records'], []);
 
   return (
     <GroupDetails
       group={group}
+      fiscalYear={fiscalYear}
```

Comment:
it's confusing, because it's not one year against multiple years.
Can you leave previous name as it was, `fiscalYears`, and call the new one something like as `currentFiscalYears` or `fiscalYearsRecords`?

Reply:
Updated